lib/puma/dsl.rb

```diff
@@ -46,6 +46,14 @@ def self.ssl_bind_str(host, port, opts)
         else ''
         end
 
+      if ['127.0.0.1', 'localhost'].include?(host)
```

Comment:
Needs ipv6 detection too. I think we do this somewhere else (`const.rb`? dont rememebr)